### DISCUSSION TITLE
[Projects Beta] Make issue #, created and last commented on date and # of comments available for view

### DISCUSSION THREAD

**Discussion by manuelap-msft** [community/community#11372] [Projects and Issues]
Hello,

love projects beta! For our project, we would find it useful to have the created and last commented on date, issue number and number of comments available as fields to add to the project view.

This would be useful for us during bug triage - e.g. I remember I replied yesterday so if the last commented on date is newer someone followd up for me to check etc.

Thank you!

**Comment by glasser** [comment]
Not to be rude, but I literally cannot believe that there is no way to view the issue number inside projects. Like, I am still searching every bit of the interface to find the setting I'm missing.

**Reply by undermensch** [reply]
Only way currently seems to be to mouse over title and look at link which ends with the issue number...not a very convenient or easy way to see issue number
